docs/java-your-first-app.md

```diff
@@ -0,0 +1,163 @@
+
+<!--SNIPSTART java-project-template-workflow-implementation-->
+<!--SNIPEND-->
+
+### Initiate transfer
+
+Let's tell the server that we want this Workflow code to execute. Make sure the [Temporal server](/docs/install-temporal-server) is running in a terminal, and then from within IntelliJ run the InitiateMoneyTransfer class or from the project root run the following command:
+
+```
+./gradlew initiateTransfer
+```
+
+The two ways to initiate a Workflow with Temporal is either via the SDK or via the CLI. For this tutorial we are focusing on using the SDK to start the Workflow, which is how most Workflows would be initiated in a production environment. Here is the code we just ran to start the Workflow:
+
+<!--SNIPSTART java-project-template-workflow-initiator-->
+<!--SNIPEND-->
+
+The call to the Temporal service can be done synchronously or asynchronously and here we do it asynchronously. You will see the program run, tell you the transaction is processing, and exit.
+
+### State visibility
+
+OK, now let's check out one of the really cool value propositions that Temporal offers: application state visibility. Visit the [Temporal Web UI](localhost:8088) where you will see your Workflow listed.
+
+![Temproal web UI](/img/docs/temporal-web-ui-transfer-money-workflow.png)
+
+Click the RunId. Here we can view everything we want to know about the state of the execution of the code we told the server to run, such as what parameter values it was given, timeout configurations, scheduled retries, number of attempts, stack traceable errors, and more.
+
+So, the Workflow is running, but why hasn't the Workflow and Activity code executed yet? Click on the Task Queue name to view active "Pollers" registered to handle these Tasks. The list will be empty. There are no Workers polling the Task Queue!
+
+### The Worker
+
+As mentioned earlier, a special process called a "Worker" is responsible for actually executing pieces of Workflow code. This is what our Worker looks like:
+
+<!--SNIPSTART java-project-template-worker-->
+<!--SNIPEND-->
+
+Notice that the Worker will listen to the same Task Queue that the Workflow and Activity tasks are sent to. This is called "Task routing", and is a built-in mechanism for load balancing.
+
+<!--SNIPSTART java-project-template-shared-constants-->
+<!--SNIPEND-->
+
+Let's start the Worker. From within IntelliJ run the TransferMoneyWorkfer class, or from the project root, run the following command:
+
+```
+./gradlew startWorker
+```
+
+When the Worker starts it begins polling the Task Queue. The first Task it finds is the Workflow Task, which it executes. Executing the Workflow causes the Activity tasks to be sent to the Task Queue as well. The Worker then pulls each of the Activity Tasks in their respective order from the Task Queue and executes the corresponding Activities.
+
+Congratulations, you have run a Temporal Workflow application!
```

Comment:
Maybe check the UI again here?

Reply:
Workflow is removed from the UI when it is complete - so there would not be anything there.